### PR TITLE
{astro,math,phys}[foss/2020a] PlasmaPy v0.3.1, mpmath v1.1.0, astropy v4.0.1 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/mpmath/mpmath-1.1.0-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Author: Adam Huffman
+# adam.huffman@crick.ac.uk
+# The Francis Crick Institute
+
+easyblock = 'PythonPackage'
+
+name = 'mpmath'
+version = '1.1.0'
```
Comment (smoors) [inline easybuild/easyconfigs/m/mpmath/mpmath-1.1.0-GCCcore-9.3.0.eb:10]: @SimonPinches please add versionsuffix (also to the file name):
```suggestion
version = '1.1.0'
versionsuffix = '-Python-%(pyver)s'
```

**Review by smoors — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/m/mpmath/mpmath-1.1.0-GCCcore-9.3.0.eb (hunk 1 above)
Comment (SimonPinches) [reply easybuild/easyconfigs/m/mpmath/mpmath-1.1.0-GCCcore-9.3.0.eb]: @smoors I've fixed this in the latest commit.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node314.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/4aa0daf2417eea8392114da9b5f6dd46 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @SimonPinches!

**Pull request merged**